cmd/clusterctl/client/cluster/mover.go

```diff
@@ -493,6 +652,52 @@ func (o *objectMover) createGroup(group moveGroup, toProxy Proxy) error {
 	return nil
 }
 
+func (o *objectMover) backupGroup(group moveGroup, directory string) error {
+	createTargetObjectBackoff := newWriteBackoff()
+	errList := []error{}
+	for i := range group {
+		nodeToCreate := group[i]
```

Comment:
```suggestion
	for _, nodeToGet := range group {
```
similar in `restoreGroup`

Reply:
Good catch - ended up doing a bit more refactoring here. Also did the suggested code optimization for `createGroup`. Also renamed the object backoffs so they make more sense in the scope of `backup` and `restore`